README.md

```diff
@@ -26,6 +26,8 @@ The sampleApps folder contains sample APKs that you can test against QARK
 - python 2.7.6
 - JRE 1.6+ (preferably 1.7+)
 - OSX or RHEL6.6 (Others may work, but not fully tested)
+- On ubuntu, lib32z1 lib32ncurses5 lib32stdc++6 (can be installed with apt-get)
```

Comment:
Let's do something that's a bit more verbose with this. Something like:

For Debian/Ubuntu/Mint derivative Linux distros, install `lib32z1 `, `lib32ncurses5 `, and `lib32stdc++6` packages. You can install these through the package manager with:

    $ sudo apt-get install lib32z1 lib32ncurses5 lib32stdc++6 

You'll also need some Python packages which you can install with pip.

    $ pip install -r requirements.txt